doc/source/whatsnew/v0.19.0.rst

```diff
@@ -706,7 +706,7 @@ Previously, this worked the same as comparison operators (see above).
 A ``Series`` will now correctly promote its dtype for assignment with incompat values to the current dtype (:issue:`13234`)
 
 
-.. ipython:: python
+.. code-block:: ipython
```

Comment:
so how does line 726 work? (as there is no s)? you would need to change all of the dependent blocks